### ISSUE TITLE
cx-flow failing with error code 10 , while creating the jira tickets , even when there are high vulnerabilities in the report , its showing 0 in the logs

### ISSUE THREAD

**Issue by Mohit-Sharma-cloud** [checkmarx-ltd/cx-flow#1218]
### Description

> Provide a description of the issue

### Reproduction

> Detail the steps taken to reproduce the issue
> 
> Where applicable, please include (exclude sensitive information):
>
> - Code of Files to reproduce the issue
> - Log files
> - Application settings
> - Screenshots

### Environment Details

> Provide any information relating to the environment the issue was identified in - include applicable version and additional runtime information (include OS or other underlying infrastructure)

**Comment by Mohit-Sharma-cloud** [comment]
We are using the latest cxflow.1.6.39-java11.jar  and getting this error
![cx-flow_app_config](https://user-images.githubusercontent.com/53616608/233115300-2de98734-8e6b-422c-af75-db4b96b20cc1.png)
![cxflow-1 6 39-java11_error](https://user-images.githubusercontent.com/53616608/233115313-f7efca80-f8f1-4d51-90bf-fcf2e2d76272.png)

**Comment by Mohit-Sharma-cloud** [comment]
Hi Team,

Awaiting for your reply.

Regards,

**Comment by satyamchaurasiapersistent** [comment]
Hi @Mohit-Sharma-cloud this issue has been resolved in our latest Release 1.6.40 please test and let me know results.
I am closing this issue in case you have any concern you can reopen this issue.

**Comment by Mohit-Sharma-cloud** [comment]
Hi @satyamchaurasiapersistent , I have tried the latest 1.6.40-java11.jar  its giving the successful result at the end but its giving a weird error while creating the ScanSetting, it does not break the process but its coming up every time. I have tested it multiple times  and getting the same  error, I am attaching the SS for the error.
PFA.

![cxflow-1 6 40-java11_error](https://user-images.githubusercontent.com/53616608/234817495-ac3ea3e0-c592-4fe6-84ab-fc166e2c8fe1.png)
